distributions/kfdef/kfctl_aws.v1.3.0.yaml

```diff
@@ -22,18 +22,13 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: common/cert-manager/cert-manager-kube-system-resources/base
```

Comment:
since we are not supporting using kfctl for 1.3 I think we should leave this file unchanged

Reply:
This file was already published by PatricXYZ when 1.3.0 was released. So I am just keeping it up to date with the branch i.e. 1.3.1.

I suggest either we remove it completely or keep this working version

Reply:
Lets remove it.

Reply:
Lets remove this and all kfctl references

Reply:
@rrrkharse @goswamig done, please take another look